code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -36,6 +34,13 @@
 		animation_math = new/list()
 	if(!door_anim_time == 0 && !animation_math["[door_anim_time]-[door_anim_angle]-[azimuth_angle_2]-[radius_2]-[door_hinge]"])
 		animation_list()
+		AddElement(/datum/element/climbable, climb_time = crate_climb_time * 0.5, climb_stun = 0)
+	else
+		AddElement(/datum/element/climbable, climb_time = crate_climb_time, climb_stun = 0)
```

Comment:
Having this logic tied to the door animation time seems incredibly strange to me.

Reply:
Agreed. I dont like the way tg implemented it, ill just comment it out. People can just climb on an open/closed crate at the same speed.